packages/medusa/src/models/batch-job.ts

```diff
@@ -18,10 +19,17 @@ export class BatchJob extends SoftDeletableEntity {
   created_by_user: User
 
   @DbAwareColumn({ type: "jsonb", nullable: true })
-  context: { retry_count?: number; max_retry?: number } & Record<string, unknown>
+  context: {
+    retry_count?: number
+    max_retry?: number
+  } & RequestQueryFields & Record<string, unknown>
 
   @DbAwareColumn({ type: "jsonb", nullable: true })
-  result: Record<string, unknown>
+  result: {
+    count?: number
+    advancementCount?: number
```

Comment:
todo: Modify to follow general convention
```suggestion
    advancement_count?: number
```